libs/workspace-manager/src/lib/features/workspace-manager/workspace-manager.component.scss

```diff
@@ -0,0 +1,46 @@
+@use '@angular/material' as mat;
+@use 'material/colors' as colors;
+
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.workspace-manager-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 150px;
+  background-color: mat.get-color-from-palette(colors.$primary);
+  color: mat.get-color-from-palette(colors.$primary, '300-contrast');
+  padding: 0 10px;
+
+  h1 {
+    display: flex;
+    align-items: center;
+    gap: 5px;
+    font-size: 18px;
+
+    img {
+      width: 50px;
+      height: 50px;
+    }
+  }
+
+  nav {
```

Comment:
you should remove most of these styles after you'll use material's nav list

Reply:
This is the mat nav list: 

https://user-images.githubusercontent.com/59396491/221976847-54288631-e309-4b11-af0b-ef58b310553c.mov


This my custom styled nav tag: 

https://user-images.githubusercontent.com/59396491/221977087-fb2aa464-9208-4e2f-b04e-9d54d06b1295.mov

Ignoring the fact that mine had icons and the mat nav didnt I think I like the looks of the material buttons and the colors of the custom styled  regular nav tag. What do you think?

Reply:
you should use "tabs" not "nav-list"
https://material.angular.io/components/tabs/overview

Reply:
after seeing how it looks, go with tabs as @Danieliverant said

Reply:
done